instance/models/server.py

```diff
@@ -232,44 +311,51 @@ def public_ip(self):
 
         return public_addr['addr']
 
-    def update_status(self, provisioning=False, rebooting=False, failed=None):
+    def update_status(self):
         """
         Refresh the status by querying the openstack server via nova
```

Comment:
Yes, the docstring is correct - the `os_server` property queries the OpenStack API, and then this code checks the resulting `status`.